### ISSUE TITLE
Laravel Mix Issue

### ISSUE THREAD

**Issue by HashandSalt** [Va1/browser-sync-webpack-plugin#66]
Version 2 seems to break with Laravel Mix. Webpack says its building but fails to launch the browser, and browsersync. Webpack says its watching files and then almost immediately says its done and drops back to terminal prompt.

This breaks:
```
"devDependencies": {
    "browser-sync": "^2.23.6",
    "browser-sync-webpack-plugin": "^2.0.1",
    "cross-env": "^5.0.5",
    "laravel-mix": "^2.1.7"
  },
```
This works

```
  "devDependencies": {
    "browser-sync": "^2.23.6",
    "browser-sync-webpack-plugin": "^1.2.0",
    "cross-env": "^5.0.5",
    "laravel-mix": "^2.1.7"
  },
```

Any idea what the problem could be? Trouble is, Mix auto installs the most recent version of `browser-sync-webpack-plugin` so i'm sure I won't be the only person to run into it.

I'm using it standalone, not in a Laravel project.

**Comment by alesvaupotic** [comment]
I can confirm the issue. Just did a npm update today and it stopped working.

**Comment by malinushj** [comment]
@HashandSalt @alesvaupotic what versions of node are you using? I'll do some tests with laravel-mix when I get the chance

**Comment by HashandSalt** [comment]
Phew! glad its not just us. Running on Node 8.9.1 on a Mac (High Sierra).

**Comment by alesvaupotic** [comment]
Ubuntu 16.04.4 LTS

`ales@r35:/var/www/$ node -v`
v8.10.0
`ales@r35:/var/www/$ npm -v`
5.7.1

Not sure about which version was last used, I just did an update as part of my weekly routine.

**Comment by HashandSalt** [comment]
if it helps with testing theres a simple [repo over](https://github.com/Va1/browser-sync-webpack-plugin/issues/66#issuecomment-373980499) here that the issue is present in.

It's also affecting [Slate](https://slateengine.com), our open source framework, so hoping the issue is resolved soon.

**Comment by alesvaupotic** [comment]
Went back and found out the last working version for me is 2.0.1.

**Comment by malinushj** [comment]
@HashandSalt Alright, I figured node version wasn't the issue but had to check. The repo link you provided is a link to this thread, can you provide the right link? Thanks

**Comment by HashandSalt** [comment]
Oh whoops, sorry, ive fixed the link.

**Comment by alesvaupotic** [comment]
Thanks, my fix exactly. I hardcoded it also to 2.0.1 and it's been fine since.

**Comment by malinushj** [comment]
@HashandSalt @alesvaupotic I fixed it here https://github.com/Va1/browser-sync-webpack-plugin/pull/67
Just needs to merged.

**Comment by Va1** [comment]
fix is shipped with version 2.2.2. thanks guyz

**Comment by alesvaupotic** [comment]
Works like a charm. Thanks!
